packages/contracts/contracts/test-helpers/FailingReceiver.sol

```diff
@@ -1,13 +1,7 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.8.9;
 
-/**
```

Comment:
I don't think natspec is really necessary for these sort of micro test helpers

Reply:
I don't think it hurts necessarily but not super opinionated